package.json

```diff
@@ -54,6 +54,7 @@
     "d3-time-format": "2.2.3",
     "flexsearch": "0.6.30",
     "page": "1.11.6",
+    "throttle-debounce": "^2.2.1",
```

Comment:
Please pin the dependency by running `npm install -E throttle-debounce`.

Reply:
More information on why we've been pinning dependencies:

https://docs.renovatebot.com/dependency-pinning/